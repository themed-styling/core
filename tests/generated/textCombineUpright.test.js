import 'jest-styled-components'

import { textCombineUpright } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(textCombineUpright, 'textCombineUpright')
