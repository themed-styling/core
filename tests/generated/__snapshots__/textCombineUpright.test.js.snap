// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textCombineUpright renders 0 1`] = `
Array [
  .c0 {
  text-combine-upright: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 0;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders function 1`] = `
Array [
  .c0 {
  text-combine-upright: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: has;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: has;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders invalid theme path 1`] = `
Array [
  .c0 {
  text-combine-upright: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders negative number 1`] = `
Array [
  .c0 {
  text-combine-upright: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  text-combine-upright: July;
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: July;
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    text-combine-upright: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    text-combine-upright: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    text-combine-upright: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    text-combine-upright: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  text-combine-upright: July;
}

@media May {
  .c0 {
    text-combine-upright: August;
  }
}

@media April {
  .c0 {
    text-combine-upright: September;
  }
}

@media January {
  .c0 {
    text-combine-upright: November;
  }
}

@media April {
  .c0 {
    text-combine-upright: August;
  }
}

@media June {
  .c0 {
    text-combine-upright: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: July;
}

@media May {
  .c0 {
    text-combine-upright: August;
  }
}

@media April {
  .c0 {
    text-combine-upright: September;
  }
}

@media January {
  .c0 {
    text-combine-upright: November;
  }
}

@media April {
  .c0 {
    text-combine-upright: August;
  }
}

@media June {
  .c0 {
    text-combine-upright: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-combine-upright: November;
  }
}

@media April {
  .c0 {
    text-combine-upright: January;
  }
}

@media January {
  .c0 {
    text-combine-upright: December;
  }
}

@media April {
  .c0 {
    text-combine-upright: April;
  }
}

@media June {
  .c0 {
    text-combine-upright: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-combine-upright: November;
  }
}

@media April {
  .c0 {
    text-combine-upright: January;
  }
}

@media January {
  .c0 {
    text-combine-upright: December;
  }
}

@media April {
  .c0 {
    text-combine-upright: April;
  }
}

@media June {
  .c0 {
    text-combine-upright: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    text-combine-upright: July;
  }
}

@media April {
  .c0 {
    text-combine-upright: August;
  }
}

@media January {
  .c0 {
    text-combine-upright: September;
  }
}

@media April {
  .c0 {
    text-combine-upright: November;
  }
}

@media June {
  .c0 {
    text-combine-upright: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    text-combine-upright: July;
  }
}

@media April {
  .c0 {
    text-combine-upright: August;
  }
}

@media January {
  .c0 {
    text-combine-upright: September;
  }
}

@media April {
  .c0 {
    text-combine-upright: November;
  }
}

@media June {
  .c0 {
    text-combine-upright: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-combine-upright: January;
  }
}

@media April {
  .c0 {
    text-combine-upright: December;
  }
}

@media January {
  .c0 {
    text-combine-upright: April;
  }
}

@media April {
  .c0 {
    text-combine-upright: May;
  }
}

@media June {
  .c0 {
    text-combine-upright: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    text-combine-upright: January;
  }
}

@media April {
  .c0 {
    text-combine-upright: December;
  }
}

@media January {
  .c0 {
    text-combine-upright: April;
  }
}

@media April {
  .c0 {
    text-combine-upright: May;
  }
}

@media June {
  .c0 {
    text-combine-upright: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders postitive number 1`] = `
Array [
  .c0 {
  text-combine-upright: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders string 1`] = `
Array [
  .c0 {
  text-combine-upright: July;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: July;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: June;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: June;
}

<div
    className="c0"
  />,
]
`;

exports[`textCombineUpright renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`textCombineUpright renders valid theme path 1`] = `
Array [
  .c0 {
  text-combine-upright: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  text-combine-upright: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
